src/robot/client.ts

```diff
@@ -813,4 +813,23 @@ export class RobotClient extends EventDispatcher implements Robot {
     });
     return stream;
   }
+
+  // MODULES
+
+  async restartModule(moduleId?: string, moduleName?: string) {
+    const { robotService } = this;
+    const request = new proto.RestartModuleRequest();
+
+    if (moduleId !== undefined) {
+      request.setModuleId(moduleId);
+    }
+    if (moduleName !== undefined) {
+      request.setModuleName(moduleName);
+    }
+    const response = await promisify<
+      proto.RestartModuleRequest,
+      proto.RestartModuleResponse
+    >(robotService.restartModule.bind(robotService), request);
+    return response.toObject();
```

Comment:
Probably don't need to actually return anything since the return type doesn't include any data

Reply:
Yeah makes sense. If we add fields to the response in the future, then we can return it.